spec/features/story_spec.rb

```diff
@@ -0,0 +1,46 @@
+# typed: false
+
+require "rails_helper"
+
+# uses page.driver.post because we're not running a full js engine,
+# so the call can't just be click_on('delete'), etc.
+
+RSpec.feature "Stories" do
+  let(:user) { create(:user) }
+  let(:story) { create(:story, user: user) }
+  let!(:inactive_user) { create(:user, :inactive) }
+  let!(:story_without_user) { create(:story) }
+
+  before(:each) { stub_login_as user }
+
+  feature "disowning stories" do
+    scenario "disowning a story" do
+      page.driver.post "/stories/#{story.short_id}/disown"
+      story.reload
+      expect(story.user).to eq(inactive_user)
+
+      visit "/s/#{story.short_id}"
+      expect(page).to have_content("inactive-user")
+    end
+
+    scenario "shows disown in story page" do
+      visit "/s/#{story.short_id}"
+      expect(page).to have_link("disown")
+    end
+
+    scenario "does not show disown in active stories page" do
+      visit "/active"
+      expect(page).not_to have_link("disown")
```

Comment:
How does this pass? I don't see the call to `if single_story` in the template that I would've expected.

Reply:
because the only story wasn't associated to the current user, even though it should not shown the `disowned` link due the current route, it was not a proper way to validate it.

I modified the test stories to have one being candidate to `disowning` (if we were on the correct path)